File: python_programming/Practices/fizz_buzz.py
#CC 6th FizzBuzz

y= 1 

while y < 51: 
    if y % 3 == 0: 
        print("Fizz")
    elif y % 5 == 0: 
        print("Buzz") 
    elif y % 3 == 0 and y %5 == 0: 
        print("FizzBuzz") 
    else:
        print(y) 
    y += 1 

File: c_programming/c_programming/practices/fizzbuzz.c
//CC 6th fizzbuzz

#include <stdio.h>

int main() {
    int y = 1;

    while (y < 51) {
        if (y % 3 == 0 && y % 5 == 0) {
            printf("FizzBuzz\n");
        } else if (y % 3 == 0) {
            printf("Fizz\n");
        } else if (y % 5 == 0) {
            printf("Buzz\n");
        } else {
            printf("%d\n", y);
        }
        y++;
    }

    return 0;
}
